docinfo.html

```diff
@@ -44,8 +44,4 @@
   padding-top: 0;
   padding-bottom: 0;
 }
-
-table.dfdexample tr {
-  : 50%;
```

Comment:
I put this in because without it the dfd examples were far too large. Did you do something else to keep them at a reasonable size?

Reply:
This part doesn't seem to be syntactically correct thus it's a no-op.